lib/zendesk_apps_tools/server.rb

```diff
@@ -17,6 +17,7 @@ class Server < Sinatra::Base
           last_mtime = curr_mtime
         end
       end
+      settings.parameters.merge!(locale: params['locale'])
```

Comment:
@pdeuter I think he is pointing out the app parameter `locale` could potentially be overwritten by the URL parameter being sent along from a classic helper.  I'm not sure if this is the same issue you are referring to.